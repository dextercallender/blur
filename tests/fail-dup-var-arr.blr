void main() {
    int a = 7;
    int[] a = [5, 341, 4];
}