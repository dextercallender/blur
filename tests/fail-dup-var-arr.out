Fatal error: exception Failure("Duplicate variable a")